app/lib/l10n/app_de.arb

```diff
@@ -0,0 +1,35 @@
+{
+    "language": "Deutsch",
+    "all": "alle",
+    "noMessages": "Noch keine Gespräche",
+    "loading": "Wird geladen",
+    "news": "Nachrichten",
+    "index1": "Index 0: Nachrichten",
+    "index2": "Index 1: Häufig gestellte Fragen",
+    "index4": "Index 4: Gemeinschaft",
+    "faq" : "Häufig gestellte Fragen",
+    "loadingConvo":  "Konversationen laden",
+    "index3" : "Index 3: Chat: Warten",
+    "guest" : "Gast",
+    "login" : "Einloggen",
+    "tasks": "Aufgaben",
+    "events": "Veranstaltungen",
+    "images": "Bilder",
+    "videos": "Videos",
+    "documents": "Unterlagen",
+    "resources": "Ressourcen",
+    "budget": "Budget",
+    "news": "Nachrichten",
+    "community": "Gemeinschaft",
+    "chat": "Plaudern",
```

Comment:
We'll keep a few of them in englisch, because it's cool that way:

```suggestion
    "login" : "Einloggen",
    "tasks": "Aufgaben",
    "events": "Events",
    "images": "Bilder",
    "videos": "Videos",
    "documents": "Unterlagen",
    "resources": "Ressourcen",
    "budget": "Budget",
    "news": "News",
    "community": "Community",
    "chat": "Chat",
```